onnx/onnx.in.proto

```diff
@@ -100,7 +100,12 @@ enum Version {
   //   method and training algorithm. The execution of TrainingInfoProto
   //   can modify the values of mutable variables.
   // - Implicitly add inference graph into each TrainingInfoProto's algorithm.
-  IR_VERSION = 0x0000000000000007;
+  IR_VERSION_2020_5_8 = 0x0000000000000007;
+
+  // IR VERSION 8 published on <TBD>
```

Comment:
may be we can change this to published with ONNX release 1.10? This way no one will forget to update the TBD